_Classes/DeusEx/Classes/BreakableWall.uc

```diff
@@ -7,7 +7,7 @@ class BreakableWall expands DeusExMover;
 //Simply pull out a melee weapon
 function bool DoLeftFrob(DeusExPlayer frobber)
 {
-    frobber.PutInHand(frobber.lastMeleeWeapon);
+    frobber.GetMeleePriority(minDamageThreshold);
```

Comment:
Rename to `SelectMeleePriority` or `SelectPriorityMeleeWeapon`. `GetPriority` implies it's returning a value, or an information function.